public/js/index.js

```diff
@@ -117,261 +122,247 @@ function lessonHandler(e) {
 	}
 }
 
-function handleViewClick(lesson) {
-	const title = lesson.querySelector('.lesson-card-title').innerText;
-	const content = lesson.querySelector('.lesson-card-content').innerHTML;
-
-	modalLessonTitle.innerText = title;
-	modalLessonContent.innerHTML = content;
-	modalLesson.removeAttribute('hidden');
-	overlay.removeAttribute('hidden');
-	overlay.classList.add('dark');
-}
-
-function lessonHelper({varName, eventListener, classList, attribute, textContent, innerHTML, id}) {
-	if(eventListener){
-		varName.addEventListener(Object.keys(eventListener)[0], Object.values(eventListener)[0]);
+function lessonHelper({
+	varName,
+	eventListener,
+	classList,
+	attribute,
+	textContent,
+	innerHTML,
+	id,
+}) {
+	if (eventListener) {
+		varName.addEventListener(
+			Object.keys(eventListener)[0],
+			Object.values(eventListener)[0],
+		);
 	}
-	if(classList){
-		for(let i = 0; i < classList.length; i++){
+	if (classList) {
+		for (let i = 0; i < classList.length; i++) {
 			varName.classList.add(classList[i]);
 		}
 	}
-	if(attribute){
-		for(let i = 0; i < attribute.length; i++){
-			varName.setAttribute(Object.keys(attribute[i])[0], Object.values(attribute[i])[0]);
+	if (attribute) {
+		for (let i = 0; i < attribute.length; i++) {
+			varName.setAttribute(
+				Object.keys(attribute[i])[0],
+				Object.values(attribute[i])[0],
+			);
 		}
 	}
-	if(textContent){
+	if (textContent) {
 		varName.textContent = textContent;
 	}
-	if(innerHTML){
+	if (innerHTML) {
 		varName.innerHTML = innerHTML;
 	}
-	if(id){
+	if (id) {
 		varName.id = id;
 	}
-	return varName
+	return varName;
 }
 
-function renderLessons() {
+function handleViewClick(lesson) {
+	const title = lesson.querySelector('.lesson-card-title').innerText;
+	const content = lesson.querySelector('.lesson-card-content').innerHTML;
+
+	modalLessonTitle.innerText = title;
+	modalLessonContent.innerHTML = content;
+	modalLesson.removeAttribute('hidden');
+	overlay.removeAttribute('hidden');
+	overlay.classList.add('dark');
+}
+
+async function renderLessons() {
 	handleClear();
 	// render lesson cards
-	return db
+	const snapshot = await db
 		.collection('users')
 		.doc(firebase.auth().currentUser.uid)
-		.get()
-		.then((snapshot) => {
-			if (lessonsContainer.childElementCount) {
-				lessonsContainer.innerHTML = '';
-			}
-
-			if (snapshot.data() && snapshot.data().lessons.length) {
-				snapshot.data().lessons.forEach(({ title, content, id }) => {
-					const lessonCard = lessonHelper({
-						varName: document.createElement('div'),
```

Comment:
One last thing 😛  can you add these `lessonHelper` function calls back in please. I think these may have been removed by mistake too. Then we are good to go 👍